Source/PlayerEx.cpp

```diff
@@ -125,11 +125,7 @@ void PlayerEx::SetSatelliteCount(map_id objectType, int count)
 
 int PlayerEx::GetColorNumber()
 {
-	if (!isInited) {
-		return HFLNOTINITED;
-	}
-
-	return playerArray[playerNum].colorNumber;
+	return isInited ? playerArray[playerNum].colorNumber : HFLNOTINITED;

```

Comment:
I don't have a problem with this shorter syntax. I just want to point out that if we change to throwing exceptions, we will effectively be undoing this change. The old return statement will be used, and the removed `if` block will be changed to the throw block/function call.

Reply:
I wanted to use the ternary operator some.